bokeh/models/plots.py

```diff
@@ -44,18 +44,8 @@
 from ..core.property_mixins import ScalarFillProps, ScalarLineProps
 from ..core.query import find
 from ..core.validation import error, warning
-from ..core.validation.errors import (
-    BAD_EXTRA_RANGE_NAME,
-    INCOMPATIBLE_SCALE_AND_RANGE,
-    REQUIRED_RANGE,
-    REQUIRED_SCALE,
-)
-from ..core.validation.warnings import (
-    FIXED_HEIGHT_POLICY,
-    FIXED_SIZING_MODE,
-    FIXED_WIDTH_POLICY,
-    MISSING_RENDERERS,
-)
+from ..core.validation.errors import BAD_EXTRA_RANGE_NAME, INCOMPATIBLE_SCALE_AND_RANGE, REQUIRED_RANGE, REQUIRED_SCALE
+from ..core.validation.warnings import FIXED_HEIGHT_POLICY, FIXED_SIZING_MODE, FIXED_WIDTH_POLICY, MISSING_RENDERERS
```

Comment:
What's the threshold for single vs multiple lines? Here the original was more readable, but then there are many other changes were a simple single line is blown out of proportion.

Reply:
So here are the options as far as I understand them

* Use `force_grid_wrap` to set the threshold at a *number of imports*. This is currently done, with value 5. We could lower it to 4 or even 2 if we just want it to always be mulit-line
* Instead, set the line-length to something lower, e.g. 100, tho this seems weird to be contradicting with our other linter, and can lead to ~bad results (IMO) on indented imports  
* There is also evidently a "wrap length" which can be lower than the line length, but I had mixed results with it in brief experiments. I could experiment a bit more with it.

Reply:
I can't get `wrap_length` to do anything I understand. I think the best option is `force_grid_wrap`, do you have any thoughts on a different value?